### DISCUSSION TITLE
2006/08/02/%E4%BD%93%E9%AA%8C-windows-live

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#102] [Announcements]
# 2006/08/02/%E4%BD%93%E9%AA%8C-windows-live

今天 MSN Spaces 改名成了 Windows Live Spaces，模样也全变了。还不咋稳定。

https://blog.qizhen.xyz/2006/08/02/%E4%BD%93%E9%AA%8C-windows-live

**Comment by ruanqizhen** [comment]
之前博客下的留言：

fx
2006年08月3日 15:31 编辑
哈哈哈~~边学边看＾＾
